src/box/tuple_compare.cc

```diff
@@ -497,18 +513,30 @@ tuple_compare_field(const char *field_a, const char *field_b,
 {
 	switch (type) {
 	case FIELD_TYPE_UNSIGNED:
+	case FIELD_TYPE_UINT8:
+	case FIELD_TYPE_UINT16:
+	case FIELD_TYPE_UINT32:
+	case FIELD_TYPE_UINT64:
```

Comment:
I'm not sure it's okay to add so many cases to this switch. Let's ask @alyapunov to be sure.

Reply:
Added benchmark results to commit message.